src/NServiceBus.Core/NServiceBus.Core.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="Particular.Packaging" Version="0.8.0" PrivateAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp2.1'">
```

Comment:
There are some complexities involved, but we likely don't need most of these references for `netcoreapp2.1`.